pdc/apps/release/routers.py

```diff
@@ -14,6 +14,8 @@
 router.register(r'releases', views.ReleaseViewSet)
 router.register(r'base-products', views.BaseProductViewSet)
 router.register(r'release-types', views.ReleaseTypeViewSet, base_name='releasetype')
+router.register(r'release-group-types', views.ReleaseGroupTypeViewSet, base_name='releasegrouptypes')
```

Comment:
@ycheng-aa ; your ideal is right, and I hope we can discuss this issue in meeting.  In my opinion, when we create a instance of release-group, need to verify the Type para would have been existed in DB, So this may be useful for user. If we don't it, it would be removed.